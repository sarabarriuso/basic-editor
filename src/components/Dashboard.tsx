import Header from './Header';
import Editor from './editor';

const Dashboard: React.FC = () => {
  const DEFAULT_CLASSNAME = 'dashboard';

  return (
    <div className={DEFAULT_CLASSNAME}>
      <Header />
      <Editor />
    </div>
  );
};

export default Dashboard;
