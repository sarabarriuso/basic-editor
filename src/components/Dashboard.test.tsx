import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Dashboard from './Dashboard';

export default describe('Dashboard Tests', () => {
  it('Renders successfully', () => {
    // Arrange
    // Act
    const tree = shallow(<Dashboard />);

    //Assert
    expect(toJson(tree)).toMatchSnapshot();
  });
});
