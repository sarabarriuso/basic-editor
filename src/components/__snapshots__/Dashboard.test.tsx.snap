// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Tests Renders successfully 1`] = `
<div
  className="dashboard"
>
  <Header />
  <Editor />
</div>
`;
